scripts/stackstorm.js

```diff
@@ -302,7 +304,9 @@ module.exports = function(robot) {
           data = e.data;
         }
 
-        postDataHandler.postData(data);
+        if (data && data.message) {
```

Comment:
Is there any other envelope information that you can post in this case? If not a sense of nothing happened is left with the user.